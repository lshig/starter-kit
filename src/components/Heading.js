import PropTypes from 'prop-types';
import React from 'react';

export default function Heading({ text }) {
  return <h1>{text}</h1>;
}

Heading.propTypes = {
  text: PropTypes.string.isRequired
};
