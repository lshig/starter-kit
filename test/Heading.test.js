import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Heading from '../src/components/Heading';

describe('Heading', () => {
  describe('basic', () => {
    const wrapper = shallow(<Heading text="Foobar" />);

    test('renders', () => {
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
