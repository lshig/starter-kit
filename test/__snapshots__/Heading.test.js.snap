// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading basic renders 1`] = `
<h1>
  Foobar
</h1>
`;
